posix.mak

```diff
@@ -373,19 +373,25 @@ $(DMD) : ${DMD_DIR}
 $(DMD_REL) : ${DMD_STABLE_DIR}
 	${MAKE} --directory=${DMD_STABLE_DIR}/src -f posix.mak AUTO_BOOTSTRAP=1
 
-dmd-release : $(STD_DDOC) $(DMD_DIR) $(DMD)
-	$(MAKE) AUTO_BOOTSTRAP=1 --directory=$(DMD_DIR) -f posix.mak html \
+$(DMD_DIR)/src/ddmd/id.d: $(DMD) | $(DMD_DIR)
+	$(MAKE) AUTO_BOOTSTRAP=1 --directory=$(DMD_DIR) -f posix.mak src/ddmd/id.d
+
+$(DMD_STABLE_DIR)/src/ddmd/id.d: $(DMD_REL) | $(DMD_STABLE_DIR)
+	$(MAKE) AUTO_BOOTSTRAP=1 --directory=$(DMD_STABLE_DIR) -f posix.mak src/ddmd/id.d
```

Comment:
Apparently with the changes, DMD isn't by default built from source anymore (or at least the built isn't accessible here).
Anyhow this seems to do the trick.